components/sections/Templates/TemplateInit.tsx

```diff
@@ -9,8 +9,12 @@ export default function TemplateInit({ path }: { path: string }) {
 					scrollbarWidth: 'thin',
 				}}
 				className='overflow-x-scroll p-4'
-			>{`cargo shuttle init --from ${path}`}</pre>
-			<CopyButton code={`cargo shuttle init --from ${path}`} width={16} height={16} />
+			>{`shuttle init ${template ? `--template ${template}` : `--from ${path}`}`}</pre>
+			<CopyButton
+				code={`shuttle init ${template ? `--template ${template}` : `--from ${path}`}`}
```

Comment:
This command string can be calculated before the return statement, and inserted here.
The --from option should use `--from shuttle-hq/shuttle-examples --subfolder <path>`

Reply:
cc: @joshua-mo-143

Reply:
Amended in new PR to ivan's branch